tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py

```diff
@@ -84,3 +90,29 @@ def test_registry_by_increasing_num_of_registry_pods(self, count=3):
 
         # Validate image registry pods
         validate_registry_pod_status()
+
+        # Coverage for Bz 2128263
```

Comment:
reconfigure OpenShift Container Platform registry to use RWX, this complete step is missing